src/screens/tenants/Tenants.tsx

```diff
@@ -214,28 +239,21 @@ export default class Tenants extends BaseScreen<Props, State> {
     );
   }
 
-  private tenantCreated = async (newTenant: TenantConnection) => {
+  private tenantCreated(newTenant?: TenantConnection) {
```

Comment:
You change the syntax of this method and use it for 
`private addEditTenantClosed = async (newTenant?: TenantConnection) => {`
Consistency on your choice?

Reply:
I renamed tenantCreated into addEditTenantClosed and created a new method named tenantCreated

Reply:
addEditTenantClosed ? 2 verbs in one ? You can do better ;-)

Reply:
It's because I named the component AddEditTenantDialog ( I don't think the verb 'manage' would suits because it's wide and not very meaningful)